### PR TITLE
Tweak: Make the popup more accessible by adding a focus after opening the popup

### PR DESCRIPTION
**Pull request by KingYes**

### CODE REVIEW

File: src/js/dialogs-manager.js [hunk 1]
```diff
@@ -330,26 +336,31 @@
...
 				iframe: null,
 				closeButton: false,
 				closeButtonOptions: {
 					iconClass: parentSettings.classPrefix + '-close-button-icon',
-					attributes: {},
+					attributes: {
+						role: 'button',
+						'tabindex': 0,
+						'aria-label': 'Close',
+						href: 'javascript:void(0);',
```
Comment (kobizz) [inline src/js/dialogs-manager.js:354]:
```suggestion
```
Is it necessary?

File: src/js/dialogs-manager.js [hunk 2]
```diff
@@ -330,26 +336,31 @@
 				classes: {
 					globalPrefix: parentSettings.classPrefix,
 					prefix: parentSettings.classPrefix + '-' + widgetName,
-					preventScroll: parentSettings.classPrefix + '-prevent-scroll'
+					preventScroll: parentSettings.classPrefix + '-prevent-scroll',
```
Comment (kobizz) [inline src/js/dialogs-manager.js:339]: Did you do it across the whole file?

**Review by kobizz — CHANGES_REQUESTED** [review]

File: src/js/dialogs-manager.js (hunk 1 above)
Comment (KingYes) [reply src/js/dialogs-manager.js:354]: Yes, after we change the element to `a` it must be valued.

File: src/js/dialogs-manager.js (hunk 2 above)
Comment (KingYes) [reply src/js/dialogs-manager.js:339]: Just the current array

**Pull request merged**